builds/android/res/values-es/strings.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">EasyRPG Player</string>
+	<!-- Preferences keys, do not translate -->
+	<string name="pref_enable_vibration">PREF_ENABLE_VIBRATION</string>
+	<string name="pref_layout_transparency">PREF_LAYOUT_TRANSPARENCY</string>
+	<string name="pref_vibrate_when_sliding_direction">PREF_VIBRATE_WHEN_SLIDING</string>
+
```

Comment:
I wonder if we could remove them from this file altogether. It would make the translation interface tidier.